test/resources/stacktrace-sourcemap/test-min.min.js

```diff
@@ -1,5 +1,4 @@
-function test() {
+export function test() {
```

Comment:
You have updated this file but not the `test-min.min.js.map` file

Reply:
It was a try to minimize the file but `prettier` is automatically formatting it after commiting or pushing it :S